.travis.yml

```diff
@@ -17,9 +21,10 @@ branches:
     - /^rc-.*$/
     - /^[0-9]+\.[0-9]+\.[0-9]+.*/
 
+# TODO: Update after https://github.com/blackbaud/skyux-sdk-cli/pull/36 is released
```

Comment:
Calling this out as well, this line can be reverted to what's published after https://github.com/blackbaud/skyux-sdk-cli/pull/36 is released.